### PR TITLE
STOR-1120: Add csi idempotency flags

### PR DESCRIPTION
**Pull request by gnufied**
https://issues.redhat.com/browse/STOR-1120

### CODE REVIEW

File: assets/vsphere_features_config.yaml [hunk 1]
```diff
@@ -5,6 +5,8 @@ data:
   "online-volume-extend": "true"
   "block-volume-snapshot": "true"
   "improved-volume-topology": "false" # operator will set this value to true if cluster supports CSI topology
+  "improved-csi-idempotency": "true"
+  "block-volume-snapshot": "true"
```
Comment (bertinatto) [inline assets/vsphere_features_config.yaml:9]: Could you add comments to these flags as you add them?

Comment (gnufied) [reply assets/vsphere_features_config.yaml:9]: As in what they do?

Comment (gnufied) [reply assets/vsphere_features_config.yaml]: done.

**Review by bertinatto — COMMENTED** [review]
/lgtm

**Comment by gnufied** [comment]
Not a user facing change.

/label docs-approved
/label px-approved

**Pull request merged**